lab-11-JR/lib/cors.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
You don't need to create the cors functionality. Just install and require in the cors module from NPM.